vclib/src/commonMain/kotlin/at/asitplus/wallet/lib/agent/Validator.kt

```diff
@@ -373,23 +387,27 @@ class Validator(
         val isRevoked = checkRevocationStatus(sdJwt) == RevocationStatus.REVOKED
         if (isRevoked)
             Napier.d("verifySdJwt: revoked")
-        val rawDisclosures = input.substringAfter("~").split("~").filterNot { it.contains(".") }
-        val disclosures = rawDisclosures.associateWith {
-            SelectiveDisclosureItem.deserialize(it.decodeToByteArray(Base64UrlStrict).decodeToString()).getOrNull()
-        }.filterValues { it != null }
+
+        val disclosures = sdJwtSigned.disclosures.filterValues { it != null }
```

Comment:
please elaborate

Reply:
Left over from old parsing code, is refactored now, so that the disclosures are never null